opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -297,7 +325,7 @@ namespace Opm {
         {
             const Grid& grid = ebosSimulator_.vanguard().grid();
             if (PolymerModule::hasPlyshlog() || GET_PROP_VALUE(TypeTag, EnablePolymerMW) ) {
-                computeRepRadiusPerfLength(grid);
+                computeRepRadiusPerfLength(grid, local_deferredLogger);
```

Comment:
Should be moved inside the try block, since it can throw.

Reply:
done